Source/WebCore/page/scrolling/ScrollAnchoringController.h

```diff
@@ -39,27 +43,23 @@ enum class CandidateExaminationResult {
 class ScrollAnchoringController final : public CanMakeWeakPtr<ScrollAnchoringController> {
     WTF_MAKE_FAST_ALLOCATED;
 public:
-    explicit ScrollAnchoringController(LocalFrameView& frameView)
-        : m_frameView(frameView)
-    { }
-
+    explicit ScrollAnchoringController(ScrollableArea&);
     void invalidateAnchorElement();
     void adjustScrollPositionForAnchoring();
     void selectAnchorElement();
     void chooseAnchorElement(Document&);
     void updateAnchorElement();
-    LocalFrameView& frameView() { return m_frameView; }
 
 private:
     Element* findAnchorElementRecursive(Element*);
     CandidateExaminationResult examineCandidate(Element&);
     bool didFindPriorityCandidate(Document&);
-    ScrollOffset computeOffset(RenderObject& candidate);
+    FloatPoint computeOffsetFromOwningScroller(RenderObject& candidate);
```

Comment:
The "from owning scroller" is a bit ambiguous because the "owning scroller" should always be the ScrollableArea associated with this ScrollAnchoringController, right?

Reply:
This was intended to mean calculate the offset of this render object from the owning scroller of this ScrollAnchoringController.